rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json

```diff
@@ -90,6 +94,97 @@
         }
       }
     },
+    "disable_circulation": {
```

Comment:
Not sure that the field name is the best one. The idea is not to block any circulation operations, but to always *display* an item with a red availability dot (instead of the green dot).

Name suggestion: `negative_availability`

Typical example: the item has as base item type "Standard" and as temporary item type "In acquisition". For "in acquisition", all items are displayed with the red dot. But they can be requested for example.

Reply:
done